core/iwasm/compilation/aot_llvm.c

```diff
@@ -2003,6 +2003,29 @@ aot_destroy_comp_context(AOTCompContext *comp_ctx)
     wasm_runtime_free(comp_ctx);
 }
 
+static bool
+insert_native_symbol(AOTCompContext *comp_ctx, const char *symbol, int32 idx)
+{
+    AOTNativeSymbol *sym = wasm_runtime_malloc(sizeof(AOTNativeSymbol));
+
+    if (!sym) {
+        aot_set_last_error("alloc native symbol failed.");
+        return false;
+    }
+
+    memset(sym, 0, sizeof(AOTNativeSymbol));
+    snprintf(sym->symbol, sizeof(sym->symbol), "%s", symbol);
```

Comment:
Should we add an assert to ensure the symbol is no longer than the buffer?
``` C
bh_assert(strlen(symbol) <= sizeof(sym->symbol));
```

Reply:
OK